resolwe_bio/tests/processes/test_files_to_reads.py

```diff
@@ -9,22 +9,23 @@
 class FilesToReadsTestCase(BioProcessTestCase):
     @tag_process("basespace-file-import", "files-to-fastq-single")
     def external_test_files_fq_single(self):
+        output_folder = Path("basespace_import") / "output"
         with self.preparation_stage():
             # Token with limited scope pre-obtained from dedicated BaseSpace testing app.
             handle = Secret.objects.create_secret(
-                "9bdf059c759a429f8af52ca084130060", self.admin
+                "d23ee29d4db4454480ad89d925475913", self.admin
```

Comment:
Document why new token and where it comes from. And perhaps how one obtains it on their own.